paymentgateway/paypal/db/install.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XMLDB PATH="admin/tool/paymentplugin/paymentgateway/paypal/db" VERSION="20200915" COMMENT="XMLDB file for Moodle admin/tool/paymentplugin/paymentgateway/paypal"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../../../../../../lib/xmldb/xmldb.xsd"
+>
+  <TABLES>
+    <TABLE NAME="paymentgateway_paypal" COMMENT="Contains info about all transactions that have been made. Values and descriptions taken from https://developer.paypal.com/docs/api-basics/notifications/ipn/IPNandPDTVariables/#ipn-transaction-types">
+      <FIELDS>
+        <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="true"/>
+        <FIELD NAME="purchase_id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
```

Comment:
no underscores in field names

Reply:
A lot of the variables in the table have underscores to match the variable names provided by Paypal. Is it okay to keep those as such? Or should we work around it?